CHANGES.md

```diff
@@ -60,12 +62,12 @@
   Create a new scheduler job at `optional-components/scheduler-job-backup` which runs the `bin/birdhouse backup create` 
   command at regular intervals to ensure that the birdhouse stack's data is regularly backed up.
 
-  To configure this job you may set the following variables:
+  To configure this job you must set the following variables:
     - `SCHEDULER_JOB_BACKUP_FREQUENCY`:
       - Cron schedule when to run this scheduler job (default is `'1 1 * * *'`, at 1:01 am daily)
     - `SCHEDULER_JOB_BACKUP_ARGS`:
```

Comment:
If these var changed from "may" to "must", they could be added to `VARS` to enforce they are "must"?

Reply:
https://github.com/bird-house/birdhouse-deploy/blob/02206e1cc5cb48e22103ac9e61bf667cd96263ef/birdhouse/optional-components/scheduler-job-backup/default.env#L16-L20